### PR TITLE
feat(crm): updating precision in support case columns

### PR DESCRIPTION
**Pull request by DevMagnataur**
## Changes in this PR
Change the case value maximum number allowed

### CODE REVIEW

**Review by serenaabbott11 — CHANGES_REQUESTED** [review]

File: app/controllers/support/cases/additional_contacts_controller.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@ def new
     def create
       @case_additional_contact_form = CaseAdditionalContactForm.from_validation(validation)
       @current_case = Support::Case.find(case_additional_contact_form_params[:support_case_id]) if @current_case.blank?
-      @emails = @current_case.case_additional_contacts&.pluck(:email)
```
[the remark this answers is not in the document]
Comment (DevMagnataur) [reply app/controllers/support/cases/additional_contacts_controller.rb:17]: No

File: app/controllers/support/establishments_controller.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ def index
     def list_for_non_participating_establishment
       respond_to do |format|
         format.json do
-          render json: EstablishmentSearch.omnisearch(params[:q]).where("establishment_type not in (?)", ["Federation", "Trust", "Single-academy Trust", "Multi-academy Trust", "Umbrella trust"]).as_json(methods: %i[autocomplete_template])
+          render json: EstablishmentSearch.omnisearch(params[:q]).where("establishment_type not in (?)", ["Federation", "Trust", "Single-academy Trust", "Multi-academy Trust", "Umbrella trust", "Local authority"]).as_json(methods: %i[autocomplete_template])
```
[the remark this answers is not in the document]
Comment (DevMagnataur) [reply app/controllers/support/establishments_controller.rb:14]: no

File: app/forms/support/case_additional_contact_form_schema.rb [hunk 3]
```diff
@@ -3,8 +3,8 @@ class CaseAdditionalContactFormSchema < Dry::Validation::Contract
     include Concerns::TranslatableFormSchema
 
     params do
-      required(:first_name).value(:string)
-      required(:last_name).value(:string)
+      optional(:first_name).value(:string)
```
[the remark this answers is not in the document]
Comment (DevMagnataur) [reply app/forms/support/case_additional_contact_form_schema.rb:6]: no

File: app/models/support/case_additional_contact.rb [hunk 4]
```diff
@@ -3,7 +3,7 @@ class CaseAdditionalContact < ApplicationRecord
     belongs_to :case, class_name: "Support::Case", foreign_key: "support_case_id"
     belongs_to :organisation, class_name: "Support::Organisation", optional: true
 
-    validates :first_name, :last_name, :email, presence: true
+    validates :email, presence: true
```
[the remark this answers is not in the document]
Comment (DevMagnataur) [reply app/models/support/case_additional_contact.rb:6]: no

**Review by serenaabbott11 — APPROVED** [review]

**Pull request merged**